lib/node_modules/@stdlib/string/next-grapheme-cluster-break/lib/main.js

```diff
@@ -30,30 +32,32 @@ var getEmojiProperty = require( './get_emoji_property.js' );
 // MAIN //
 
 /**
-* Returns the next grapheme cluster break (boundary).
+* Returns the next extended grapheme cluster break in a string after a specified position.
 *
-* @private
 * @param {string} str - input string
-* @param {NonNegativeInteger} idx - position in string for starting search
+* @param {integer} idx - position
```

Comment:
If `idx` can be negative, `integer` is appropriate. However, I don't think that is the intent here. In which case, if the value must be a nonnegative integer, then the type should be `NonNegativeInteger`, as in the original file.